shopify/utils/utilities.py

```diff
@@ -47,8 +49,7 @@ def __validate_issuer(cls, decoded_payload):
 
     @classmethod
     def __validate_issuer_hostname(cls, decoded_payload):
-        hostname_pattern = r"[a-z0-9][a-z0-9-]*[a-z0-9]"
-        shop_domain_re = re.compile(r"^https://{h}\.myshopify\.com/$".format(h=hostname_pattern))
+        shop_domain_re = re.compile(r"^https://{h}\.myshopify\.com/$".format(h=HOSTNAME_PATTERN))
```

Comment:
I wonder if it would make sense to have this call `sanitize_shop_domain` and fail if we get an empty result back. That's not necessarily what I would do, I was just wondering whether that made sense.

Reply:
That's a good idea!